Casks/n/nrfutil.rb

```diff
@@ -0,0 +1,16 @@
+cask "nrfutil" do
+  version "1.0.0-60fe18a"
+  sha256 "229af1183c16cef8e18554ef3a922cb56f16eb7ee8dce76d16a0ace94e7fc6d1"
+
+  url "https://developer.nordicsemi.com/.pc-tools/nrfutil/universal-osx/nrfutil-launcher-universal-apple-darwin-#{version}"
+  name "nrfutil"
+  desc "Unified CLI utility for Nordic Semiconductor products"
+  homepage "https://www.nordicsemi.com/Products/Development-tools/nrf-util"
+
+  livecheck do
+    url "https://developer.nordicsemi.com/.pc-tools/nrfutil/universal-osx/"
+    regex(/nrfutil-launcher-universal-apple-darwin-([\d.]+-[a-z0-9]+)/)
```

Comment:
This may be better as it follows some of our typical regex conventions.  Will still wait for @samford

```suggestion
    regex(/nrfutil-launcher-universal-apple-darwin-((\d+(?:\.\d+)+)[._-]\h+)/)
```

Reply:
```suggestion
    regex(/nrfutil-launcher-universal-apple-darwin[._-]v?(\d+(?:\.\d+)+(?:[._-]\h+)?)/i)
```

I agree with @krehel's suggestion but we may want to make the hash suffix optional, just in case they drop it at some point in the future. This would allow the regex to continue matching versions, instead of failing with an `Unable to get versions` error.

Besides that, this suggestion also includes a couple other tweaks to bring it in line with our typical regex patterns.

Reply:
I like this!  Thanks @samford for the enhancements. 😄